### ISSUE TITLE
How do i unchange all the modifications?

### ISSUE THREAD

**Issue by manoelacmn** [stav121/i3wm-themer#135]
Because mine is I3-gaps instead of i3-wm,all my settings are messed up,like for example,my rofi doesn't work anymore,and mod key is now "alt" instead windows even thought in the i3's config file is supposed to be the windows key

**Comment by stav121** [comment]
Hello @manoelacmn ,

If you followed the steps described in the README.md section of the repository on how to backup your files before applying the script, you should be able to revert the changes manually or via the script.

If you didn't then I don't think there is any way to revert the changes sadly.

**Comment by manoelacmn** [comment]
Ok,i ll look into it,thx for your attention

**Comment by ryanalexmartin** [comment]
The "backup" section should really be before the section that claims to be only installing dependencies.

**Comment by stav121** [comment]
> The "backup" section should really be before the section that claims to be only installing dependencies.

Yes, you are right, maybe I should look into dockerizing it to make it completely independent.
